RecoParticleFlow/PFClusterProducer/plugins/SimMappers/RealisticHitToClusterAssociator.h

```diff
@@ -0,0 +1,288 @@
+#ifndef __RecoParticleFlow_PFClusterProducer_RealisticHitToClusterAssociator_H__
+#define __RecoParticleFlow_PFClusterProducer_RealisticHitToClusterAssociator_H__
+/////////////////////////
+// Author: Felice Pantaleo
+// Date:   30/06/2017
+// Email: felice@cern.ch
+/////////////////////////
+#include <vector>
+#include "RealisticCluster.h"
+
+namespace
+{
+//TODO: get number of layers per subdet from geometry
```

Comment:
I cannot find a method which returns the number of layers. 
Should I really use something like the following?
`auto geomBH = static_cast<const HGCalGeometry*>(geom_->getSubdetectorGeometry(DetId::Forward,ForwardSubdetector::HGCHEB));

bhOffset_+ (geomBH->topology().dddConstants()).layers(true);`